apps/browser/qml/pages/components/FavoriteGrid.qml

```diff
@@ -52,6 +59,16 @@ IconGridViewBase {
         menu: favoriteContextMenu
         openMenuOnPressAndHold: false
 
+        onMenuOpenChanged: {
```

Comment:
I'm finding that when I open the context menu on a favorite item when there's a search term, the menu opens from the bottom upwards and gets cut off by the top of the overlay (see the screenshot).
![favorite-menu](https://user-images.githubusercontent.com/1446122/110356382-de420e80-8042-11eb-8de3-0cf0f9c67c07.png)

Reply:
@llewelld I cannot repeat, the context menu does not go beyond the boundaries ...
[
![Screenshot_20210310_002](https://user-images.githubusercontent.com/78217859/110652875-d2368800-81cd-11eb-88a5-3eb892f3178a.png)
](url)